### ISSUE TITLE
[distributed] Validate running s3-bench against minio server distributed

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#2563]

**Comment by harshavardhana** [comment]
Preliminary run. 

```
$ s3-bench -payload 500000 -runs 100
2016/08/28 01:59:26 Wrote http://minio3:42001/s3-bench-us-east-1-700193575/random.dat
2016/08/28 01:59:26 Wrote 51 GB in 37m58.115464563s (21947.9657k/s)
2016/08/28 02:01:44 Read (in serial) 51 GB in 2m17.515453495s (355.0737Mb/s)
2016/08/28 02:03:08 Read (in parallel) 51 GB in 1m24.283432562s (579.3324Mb/s)
```

**Comment by harshavardhana** [comment]
```
2016/08/28 03:03:09 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:13 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:18 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:23 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:27 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:32 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:36 Wrote http://minio3:42001/s3-bench-us-east-1-916994153/random.dat
2016/08/28 03:03:41 Request failed with code 404
minio@minio1:~$ AWS_ACCESS_KEY_ID=abcd123 AWS_SECRET_ACCESS_KEY=abcd12345 s3-bench -payload 100000 -runs 10000
```

**Comment by harshavardhana** [comment]
Related to the network disconnections we see in other bugs.

**Comment by harshavardhana** [comment]
Closing this as duplicate of #2561
